### PR TITLE
Discovery mock

### LINKED ISSUE #6: Complete HTML/CSS mock: Discovery/User Search

### PR DESCRIPTION
**Pull request by leah-mayo**
closes #6

### CODE REVIEW

File: project/src/main/webapp/discovery-style.css [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    text-align: center;
+}
+
+.profile-header {
+  padding: 20px;
+  margin-top: 20px;
+}
+
+.profile-header__left-column {
+  float: left;
```
Comment (gwynethsc) [inline project/src/main/webapp/discovery-style.css:28]: I recommending looking into flexbox (https://css-tricks.com/snippets/css/a-guide-to-flexbox/) in place of floats --- I think it's a personal preference, but a lot of people consider them much easier to work with.

File: project/src/main/webapp/discovery-style.css [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+#myInput {
+  background-image: url(''); /* Add a search icon to input */
+  background-position: 10px 12px; /* Position the search icon */
+  background-repeat: no-repeat; /* Do not repeat the icon image */
+  width: 100%; /* Full-width */
+  font-size: 16px; /* Increase font-size */
+  padding: 12px 20px 12px 40px; /* Add some padding */
+  border: 1px solid #ddd; /* Add a grey border */
+  margin-bottom: 12px; /* Add some space below the input */
+}
```
Comment (gwynethsc) [inline project/src/main/webapp/discovery-style.css:60]: A lot of these comments seem to just be describing the standard function of the CSS properties --- are they all needed?

**Review by gwynethsc — APPROVED** [review]

File: project/src/main/webapp/discovery-style.css (hunk 2 above)
Comment (sperzion) [reply project/src/main/webapp/discovery-style.css:60]: +1. Recommend deleting the comments.

File: project/src/main/webapp/discovery-style.css (hunk 1 above)
Comment (leah-mayo) [reply project/src/main/webapp/discovery-style.css:28]: Ok cool, thanks!

File: project/src/main/webapp/discovery-style.css (hunk 2 above)
Comment (leah-mayo) [reply project/src/main/webapp/discovery-style.css:60]: Nope they're not needed, I'll remove them.

File: project/src/main/webapp/discovery.html [hunk 3]
```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html>
+  <body style="background c">
```
Comment (sperzion) [inline project/src/main/webapp/discovery.html]: What is background c?

**Review by sperzion — APPROVED** [review]

File: project/src/main/webapp/discovery.html (hunk 3 above)
Comment (leah-mayo) [reply project/src/main/webapp/discovery.html]: It's a style attribute that I should've deleted when I pasted from my portfolio. I'll do that now.

**Pull request merged**